cmd/vic-machine/common/utils_test.go

```diff
@@ -120,3 +119,66 @@ func TestCheckUnsupportedCharsDatastore(t *testing.T) {
 		t.Errorf("got %q, expected error for %q", err, test.S)
 	}
 }
+
+func TestCheckURLValidationOutputParity(t *testing.T) {
+	tests := []struct {
+		S     string
+		valid bool
+	}{
+		{"nfs://127.0.0.1:/my/data/here:labels", true},
```

Comment:
What about the case where there are no labels? i.e. something like `"nfs://127.0.0.1:/my/data/here"`? Would your code handle that case correctly?